src/graccarchive/graccarchive.py

```diff
@@ -304,8 +304,6 @@ def main():
                         action="store_true", dest="dev")
     args = parser.parse_args()
     config = {}
-    if not args.config:
```

Comment:
How about this? Look at an envvar, or else the working directory:
```
    if not args.config:
        os.environ.get('GRACCARCHIVE_CONFIG', 'config.toml')
```

Reply:
Sure